test/plugin/constants.rb

```diff
@@ -950,4 +970,15 @@ module Constants
       'grpc-status-details-bin' => status_details.to_proto
     }.freeze
   end
+
+  PRESERVED_KEYS_MAP = {
+    'time' => K8S_TIMESTAMP,
+    'severity' => CONTAINER_SEVERITY,
+    DEFAULT_HTTP_REQUEST_KEY => HTTP_REQUEST_MESSAGE,
+    DEFAULT_OPERATION_KEY => OPERATION_MESSAGE,
+    DEFAULT_SOURCE_LOCATION_KEY => SOURCE_LOCATION_MESSAGE,
+    DEFAULT_TRACE_KEY => TRACE,
+    DEFAULT_SPAN_ID_KEY => SPAN_ID,
+    DEFAULT_INSERT_ID_KEY => INSERT_ID
```

Comment:
[Optional] You seem to have reordered the tests in alphabetical order. May make sense to reorder these too...

Reply:
Done